pipelines/vars/forklift_plugins.yml

```diff
@@ -17,6 +17,7 @@ server_box:
         - "--enable-foreman-plugin-bootdisk"
         - "--enable-foreman-plugin-discovery"
         - "--enable-foreman-plugin-hooks"
+        - "--enable-foreman-plugin-puppet"
```

Comment:
This should also be conditional so we don't break pipes like https://ci.theforeman.org/job/foreman-plugins-2.5-rpm-test-pipeline/ and https://ci.theforeman.org/job/foreman-plugins-2.5-deb-test-pipeline/ (and older).